itenium-socks/src/app/socks/sock.component.html

```diff
@@ -38,16 +38,29 @@ <h1 style="margin-top: 25px;">{{ sock.name }}</h1>
 
         <br>
         <br>
-
-        <button type="button" class="buy-socks" (click)="buy()">
+        <p *ngIf="sock.inventory === 0" class="badge">SOLD OUT</p>
+        <button type="button" class="buy-socks" (click)="buy()" *ngIf="sock.inventory > 0">
```

Comment:
Je hoeft niet altijd een functie te gebruiken:
(is waarschijnlijk ook (persoonlijke) smaak)

```
<button (click)="showModal = true">
```